doc/howto/jupysql-duckdb.md

```diff
@@ -0,0 +1,139 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+myst:
+  html_meta:
+    description lang=en: Use JupySQL and DuckDB to query JSON files with SQL
+    keywords: jupyter, sql, jupysql, json, duckdb
+    property=og:locale: en_US
+---
+
+# Analyzing Github Data with JupySQL + DuckDB
+
+JupySQL and DuckDB have many use cases. Here, let's query the Github REST API to run some analysis using these tools. 
+
+```{code-cell} ipython3
+:tags: [remove-cell]
+from pathlib import Path
+
+token = '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'
+
+paths = ["jupyterdata.json"]
+
+for path in paths:
+    path = Path(path)
+
+    if path.exists():
+        print(f"Deleting {path}")
+        path.unlink()
+```
+
+```{code-cell} ipython3
+:tags: [hide-output]
+%pip install jupysql duckdb duckdb-engine rich --quiet
+```
+
+## Pulling from Github API
+
+First, let's pull information on repositories relating to 'Jupyter' from the Github API. It requires a token, but accessing it is very simple if you have a Github account. More information on authentication can be found [here](https://docs.github.com/en/rest/guides/getting-started-with-the-rest-api?apiVersion=2022-11-28#authenticating). Your token should replace the `token` variable. Our query will pull any repository relating to Jupyter, sorted by most to least stars.
+
+```{code-cell} ipython3
+import requests
+import json
+from pathlib import Path
+
+url = 'https://api.github.com/search/repositories?q=jupyter&sort=stars&order=desc'
+headers = {
+    'Accept': 'application/vnd.github+json',
+    'Authorization': f'Bearer {token}',
+    'X-Github-Api-Version': '2022-11-28'
+          }
+res = requests.get(url, headers=headers)
```

Comment:
```suggestion
res = requests.get(
    'https://api.github.com/search/repositories',
    params={'q': 'jupyter', 'sort': 'stars', 'order': 'desc'},
    headers={
        'Authorization': f'Bearer {token}',
        'X-Github-Api-Version': '2022-11-28'
    }
)
```

Reply:
This change seems to cause the `check-for-broken-links` CI check to fail... any way to solve this @edublancas ?

Reply:
looks like this has been fixed